integration_tests/__init__.py

```diff
@@ -0,0 +1,21 @@
+import configparser
+import os
+
+
+def setup_module(module):
+    api_key = os.getenv('JC_API_KEY')
+    assert (api_key is not None), \
+        "The environmental variable `JC_API_KEY` must contain a valid Jumpcloud API key"
+
+    # Change home dir to the integration_tests or project directory (doesn't really matter which) so that the config
+    # file for testing purposes doesn't overwrite the dev's actual config file
+    assert os.path.normpath(os.getenv('HOME')) == os.path.normpath(os.getcwd()) or os.getenv("TRAVIS"), \
+        "The environmental variable `HOME` must be set to the project directory, unless this test is running on Travis"
+    with open('.jccli.ini', 'w') as config_file:
+        config = configparser.ConfigParser()
+        config['DEFAULT'] = {'key': api_key}
```

Comment:
tests will run with the default key?  hmm, that seems kinda dangerous because i may unintentionally run the test and write/remove stuff from my jumpcloud prod account?  how about having user run with a specific profile or even a different ini file?

Reply:
I designed this with a check to avoid touching a developer's actual `.jccli.ini` config file. However, it was imperfect, because it was just checking that working directory == home directory, which doesn't guarantee that the dev has changed the `HOME` env var (which is what we want).

I just updated it in the latest revision so that it checks that the `HOME` directory path ends in `/jccli` (i.e. any dev who wants to run integration tests has to set their `HOME` env var to the project dir). Does that sufficiently assure you that it's not going to let developers shoot themselves in the foot by accidentally running integration tests with their production API key?